### ISSUE TITLE
Determine extension from String/URL

### ISSUE THREAD

**Issue by Idomo** [sendyhalim/Swime#14]
Hi,
First of all, thanks for this great helper!

I thought that if there is already a list of extensions, maybe an option to get a `MimeType` from `String`/`URL`/`NSURL` won’t be that hard-to-add functionality and I think it would be an great addition to this helper.

Hope you’ll be able to add this soon,
Ido.

**Comment by sendyhalim** [comment]
@Idomo hi thanks for the kind words! 
So you're saying `String` means getting mime type from a base64 string right, and for `URL` the code will fetch first few bytes then determine the mime type based on those bytes?
If yes: I think this functionality is better to be implemented in a different repo, the reason is to keep the package small enough so people are free to choose whether to include the "extension" package or not.

**Comment by Idomo** [comment]
No, I think you’ve misunderstood me, I didn’t mentioned to use bytes at all, only from the `String` itself, the chars from the end of it (after the dot).

‫ב-25 באוג׳ 2019, בשעה 20:15, ‏‏Sendy Halim ‏<notifications@github.com> כתב/ה:‬
